### PR TITLE
hide the hybrid management tile when on stable

### PR DESCRIPTION
**Pull request by iphands**

### CODE REVIEW

**Comment by iphands** [comment]
@karelhala 
@ryelo 

This is that thing that Mega was asking for this morning.
Added tests to cover the part I changed and switched the name of the field from active -> active.

Should work. If you prefer something else, please change it and merge if you can.
Also once merged please merge `master` -> `master-stable` and release.
kthxbye

File: src/layout/Body.js [hunk 1]
```diff
@@ -93,9 +99,16 @@ Body.defaultProps = {
 };
 
 function mapStateToProps({ technologies }) {
+    let active = technologies && technologies.activeTechnologies;
```
Comment (karelhala) [inline src/layout/Body.js]: This is too complicated, no need to use lodash's filter. If you want to make sure it won't fail on undefined arrays we can update the `mapStateToProps` function to:
```JS
function mapStateToProps({ technologies = { aciveTechnologies = [] } }) {
  return {
    technologies: technologies && technologies.aciveTechnologies.filter(({ disabled }) => !disabled)
  }
}
```

File: src/layout/Body.test.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+/*global describe, test, expect */
+
+import { mapStateToProps } from './Body';
+
+function getInput(obj) {
+    return {
+        technologies: {
+            activeTechnologies: obj
```
Comment (karelhala) [inline src/layout/Body.test.js]: This should return array with `obj` in it.
```JS
return {
  technologies: {
    activeTechnologies: [obj]
  }
}
```

File: src/layout/Body.test.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        }
+    };
+}
+
+function getOutput(obj) {
+    return { technologies: obj };
```
Comment (karelhala) [inline src/layout/Body.test.js]: Same as above, this should be array.
```JS
return { technologies: [obj] };
```

**Comment by iphands** [comment]
Works for me, can you change them then merge? And ship it to beta / stable?

**Pull request merged**